src/views/reset-password/request-password-reset.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Card, Help, InputField, LogoIcon } from '@dhis2/ui'
+import { useAlert } from '@dhis2/app-service-alerts';
+import { useCallback, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useAuthAxios } from '../../hooks'
+import styles from './reset-password.module.css'
+
+export const RequestPasswordReset = () => {
+    const navigate = useNavigate()
+    const [inputs, setInputs] = useState({
```

Comment:
Why use an object with just on property? Couldn't the code be simplified by just creating a state called email?